content/applications/services/planning.rst

```diff
@@ -0,0 +1,323 @@
+  to your needs. The following rules apply to recurring shifts:
+
+  - All fields (e.g. :guilabel:`Resource`, :guilabel:`Role`, :guilabel:`Project`) are copied from
+    the original shift except for the date, which is adjusted according to the
+    :guilabel:`Repeat Every` field.
+  - Recurrences are planned but not published.
+  - By default, planned shifts are created six months in advance, after which they are created
+    gradually. To change the time frame, go to :menuselection:`Configuration --> Settings -->
+    Recurring shifts`.
+
+- :guilabel:`Save as Template`: when this option is ticked, a shift template is created with the
+  same :guilabel:`Start and End hours`, :guilabel:`Allocated time`, :guilabel:`Role`,
+  and :guilabel:`Project`.
+- :guilabel:`Note`: additional note that is sent to the employee on shift publishing.
+- :guilabel:`Date`: choose the date and time of your shift. This is the only mandatory field when
+  creating a shift.
+- :guilabel:`Allocated time`: is calculated based on the date and the employee’s :guilabel:`Working
+  Schedule`. See more in :ref:`Shift Templates <planning/templates>`.
+
+Press :guilabel:`Publish & Save` to confirm the shift, and send the assigned employee their schedule
+by email.
+
+.. note::
+   The draft is visible on the admin planning view and can be identified by diagonal lines. The
+   employee is only notified of the shift once it's published.
+
+   Two kinds of notifications are sent to the employees depending on their account configuration:
+
+   - Employees without user accounts are redirected to a dedicated Planning portal.
+   - Employees with a user account are redirected to the back end of Odoo.
+
+.. _planning/open-shifts:
+
+Open shifts and auto planning
+-----------------------------
+
+The :guilabel:`Auto Plan` button allows you to assign **Open shifts** (shifts with no resource
+assigned) and create and assign shifts linked to Sales Orders or Project.
+
+The following features have an impact on auto planning:
+
+- **Roles**: open shifts are only assigned to resources (employees or materials) that have the
+  corresponding role assigned.
+- **Default roles**: the default role assigned to a resource is given priority over the other roles
+  they have assigned to them.
+- **Conflicts**: employees or materials cannot be assigned multiple shifts at the same time.
+- **Time off**: the employees’ time off is taken into account, as well as public holidays.
+- **Company working hours**: are taken into account when assigning shifts to employees or materials.
+- **Contracts**: employees without an active contract cannot be assigned shifts.
+
+Press :guilabel:`Publish` to confirm the schedule and notify the employees of their planning.
+
+.. _planning/switching-unassignment:
+
+Switching shifts and unassignment
+---------------------------------
+
+Two features are available to allow employees to make changes to their schedule:
+**switching shifts** and **unassignment**.
```

Comment:
**Shifting shifts** and **unassignment** allow employees to make change to their schedule.


Are these fields that we can see or click on the db? I can't find them but if so I'd put a guilabel